### DISCUSSION TITLE
User dictionary location

### DISCUSSION THREAD

**Discussion by damien-r** [gitextensions/gitextensions#9751] [Q&A] [answered]
Hi,

I added by mistake an incorrect word in my dictionary. How can I remove it from my dictionary?

I tried to search for the dictionary file, but I was unable to find it.

Thanks in advance.

**Comment by RussKie** [comment]
Maybe C:\Program Files (x86)\GitExtensions\Dictionaries ?

**Reply by damien-r** [reply]
Thanks for your quick answer. No, my incorrect word is not present in these dictionaries. In addition, the "Date modified" of the files is the same as the installation date, so they have not been modified.

**Comment by EpsilonD3lta** [comment]
I would like to know this too. I want to copy all my custom words added to dictionary to another PC. I can confirm that these custom words are not present in C:\Program Files (x86)\GitExtensions\Dictionaries.

**Comment by damien-r** [comment, marked as answer]
I found it.
By following the call path from https://github.com/gitextensions/gitextensions/blob/3d4ed902fcd13dba41d66dd1e6b918ba78eac3f5/GitUI/SpellChecker/EditNetSpell.cs#L34
I got to https://github.com/gitextensions/gitextensions/blob/3d4ed902fcd13dba41d66dd1e6b918ba78eac3f5/Externals/NetSpell.SpellChecker/Dictionary/WordDictionary.cs#L101

Therefore, the Dictionary is in folder %appdata%/NetSpell (/user.dic)
